docs/products/alloydb-omni/access-ai-models.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Access and use AI models with Aiven for AlloyDB Omni®
+sidebar_label: Access AI models
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+
+Enable and use AI models in Aiven for AlloyDB Omni® to build and deploy generative AI
+applications directly on your operational data.
+
+Aiven for AlloyDB Omni® allows you to integrate and use AI models such as OpenAI, Hugging
+Face, or Gemini.
+
+## Prerequisites
+
+- Aiven for AlloyDB Omni® service running
+- Access to the [Aiven Console](https://console.aiven.io)
+- [`psql` CLI client installed](https://www.postgresql.org/download/)
+- [`gcloud` CLI client installed](https://cloud.google.com/sdk/docs/install)
+- [Google service account credentials uploaded into Aiven for AlloyDB Omni](/docs/products/alloydb-omni/manage-credentials)
+
+## Use OpenAI
+
+Use the model by running:
+
+```sql
+defaultdb=> SELECT google_ml.predict_row('gpt-4o','{"model" : "gpt-4o", "messages" : [{"role": "user", "content": "What is Aiven?"}]}')->'choices'->0->'message'->'content';
+```
+
+## Use Gemini
+
+1. Install the `google_ml_integration` extension in your database using the `psql` CLI:
+
+   ```sql
+   CREATE EXTENSION google_ml_integration VERSION '1.3';
+   ```
```

Comment:
This step is common for all integrations. Maybe put this right after the prerequisites?